src/Microsoft.PowerShell.Commands.Utility/commands/utility/Var.cs

```diff
@@ -718,6 +744,31 @@ protected override void BeginProcessing()
             {
                 _valueIsFormalParameter = true;
             }
+
+            if (_append)
+            {
+                // create the list here and add to it if it has a value
+                // but if they have more than one name, produce an error
+                if (Name.Length != 1)
+                {
+                    ErrorRecord appendVariableError = new ErrorRecord(new InvalidOperationException(), "SetVariableAppend", ErrorCategory.InvalidOperation, Name);
+                    appendVariableError.ErrorDetails = new ErrorDetails("SetVariable");
+                    appendVariableError.ErrorDetails.RecommendedAction = "Use a single variable rather than a collection";
+                    ThrowTerminatingError(appendVariableError);
+                }
+                _valueList = new List<object>();
```

Comment:
A blank line above this would help visually separate the error case (more than one name) from the rest of code.